IDRSolutions/IDRCloudClient.cs

```diff
@@ -73,7 +74,7 @@ public IDRCloudClient(string url, int conversionTimeout = 30, int requestTimeout
         /// Set to 30s by default.</param>
```

Comment:
Documentation should be updated to match new default

I would add a suggestion, but the GitHub UI is not including line relevant lines. We need 
"... If value <= 0 then the conversion does not time out.
        /// Set to -1 by default.</param>"
to be added to the Doc Comment

Reply:
The suggested change has been updated